demo/DialogBox.tscn

```diff
@@ -0,0 +1,112 @@
+[gd_scene load_steps=9 format=2]
+
+[ext_resource path="res://addons/godot_yarn_spinner/demo/Xolonium-Regular.ttf" type="DynamicFontData" id=1]
+[ext_resource path="res://addons/godot_yarn_spinner/demo/menu_section_9patch.png" type="Texture" id=2]
+[ext_resource path="res://addons/godot_yarn_spinner/demo/simple_test.yarn" type="Script" id=3]
+[ext_resource path="res://addons/godot_yarn_spinner/demo/DialogBox.gd" type="Script" id=4]
+[ext_resource path="res://addons/godot_yarn_spinner/demo/DialogText_Label.gd" type="Script" id=5]
+[ext_resource path="res://addons/godot_yarn_spinner/demo/sweet_tooth_cake.png" type="Texture" id=6]
+[ext_resource path="res://addons/godot_yarn_spinner/demo/OptionsList.gd" type="Script" id=7]
+
+[sub_resource type="DynamicFont" id=1]
+size = 20
+font_data = ExtResource( 1 )
+
+[node name="DialogBox" type="CanvasLayer"]
+script = ExtResource( 4 )
+
+[node name="OptionsMenu" type="VBoxContainer" parent="."]
+anchor_left = 0.5
+anchor_top = 0.5
+anchor_right = 0.5
+anchor_bottom = 0.5
+margin_left = -456.0
+margin_top = -412.0
+margin_right = 448.0
+margin_bottom = 228.0
+script = ExtResource( 7 )
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="Dialog_Control" type="Control" parent="."]
+anchor_left = 0.5
+anchor_top = 1.0
+anchor_right = 0.5
+anchor_bottom = 1.0
+margin_left = -264.0
+margin_top = -136.0
+margin_right = 264.0
+margin_bottom = -20.0
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="Box_Rect9" type="NinePatchRect" parent="Dialog_Control"]
+anchor_left = 0.5
+anchor_top = 1.0
+anchor_right = 0.5
+anchor_bottom = 1.0
+margin_left = -372.5
+margin_top = -164.0
+margin_right = 372.5
+texture = ExtResource( 2 )
+patch_margin_left = 3
+patch_margin_top = 3
+patch_margin_right = 3
+patch_margin_bottom = 3
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="MarginContainer" type="MarginContainer" parent="Dialog_Control/Box_Rect9"]
+anchor_right = 1.0
+anchor_bottom = 1.0
+margin_left = 110.0
+margin_top = 24.0
+margin_right = -32.0
+margin_bottom = -24.0
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="DialogText_Label" type="RichTextLabel" parent="Dialog_Control/Box_Rect9/MarginContainer"]
+margin_right = 603.0
+margin_bottom = 116.0
+size_flags_horizontal = 3
+size_flags_vertical = 7
+custom_fonts/normal_font = SubResource( 1 )
+bbcode_text = "Blah blah blah"
```

Comment:
I believe you may have switched off "Enable BBCode by accident" :sweat_smile:

Reply:
https://media.giphy.com/media/TJawtKM6OCKkvwCIqX/giphy.gif